config/detekt/detekt.yml

```diff
@@ -590,9 +590,9 @@ style:
                 value: 'FIXME:'
             -   reason: 'Forbidden STOPSHIP todo marker in comment, please address the problem before shipping the code.'
                 value: 'STOPSHIP:'
-            # -   reason: 'Forbidden TODO todo marker in comment, please do the changes.'
-            #    value: 'TODO:'
-        allowedPatterns: ''
+            -   reason: 'TODO comment format should be: // TODO: $description'
+                value: '// TODO:'
+        allowedPatterns: '^//\s*TODO:\s+.+$'
```

Comment:
Sorry my original suggestion wasn't correct, but this also seems like the original regex. I think you want this:

`allowedPatterns: '^// TODO: .+$'`